src/edu/upenn/cit594/datamanagement/PropertyDelimitedFileReader.java

```diff
@@ -0,0 +1,57 @@
+package edu.upenn.cit594.datamanagement;
+
+import edu.upenn.cit594.data.Property;
+import edu.upenn.cit594.utils.Utils;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PropertyDelimitedFileReader extends DelimitedFileReader<List<Property>> {
+    private int zipcodeIndex;
```

Comment:
I think with a refactoring of headers/headerList as a field of DelimitedFileReader, these fields could be removed.